graphicsmagick.yaml

```diff
@@ -20,25 +20,43 @@ environment:
       - libtool
       - libwebp-dev
       - libxml2-dev
+      - mercurial
       - tiff-dev
 
 pipeline:
-  - uses: fetch
-    with:
-      expected-sha256: 2b88580732cd7e409d9e22c6116238bef4ae06fcda11451bf33d259f9cbf399f
-      uri: https://sourceforge.net/projects/graphicsmagick/files/graphicsmagick/${{package.version}}/GraphicsMagick-${{package.version}}.tar.xz
+  - runs: |
+      # Clone the repository.
+      # hg clone http://hg.code.sf.net/p/graphicsmagick/code -r ${LASTEST_COMMIT}"
+      # Fetching commit instead of tar.gz, for security.
+      hg clone http://hg.code.sf.net/p/graphicsmagick/code graphicsmagick -r eff5b40bc7702651ceffbe0eeb26d5cd278abf95
 
   - uses: autoconf/configure
     with:
+      dir: graphicsmagick
       opts: |
-        --with-quantum-depth=16 \
+        --prefix=/usr \
+        --sysconfdir=/etc \
+        --mandir=/usr/share/man \
+        --infodir=/usr/share/info \
+        --localstatedir=/var \
+        --disable-openmp \
+        --disable-static \
+        --enable-shared \
         --with-gs-font-dir=/usr/share/fonts/Type1 \
+        --with-modules \
+        --with-perl \
+        --with-quantum-depth=16 \
+        --with-threads \
         --with-ttf \
-        --with-webp
+        --with-webp \
```

Comment:
nit: while this works, we should get get rid of the newline character

Reply:
Done, thanks RJ